tests/jerry/func-decl.js

```diff
@@ -23,3 +23,27 @@ assert ((function() {
     }
     return f();
 })() === 'bar');
+
+function check_syntax_error (s) {
+  try {
+    eval (s);
+    assert (false);
+  }
+  catch (e) {
+    assert (e instanceof SyntaxError);
+  }
+}
+
```

Comment:
Would it be possible to create a test case which fails (as expected) with this patch and passes (incorrectly) without it?